src/static/partials/add-workshop.html

```diff
@@ -34,27 +39,33 @@
       <label class="col-md-4 control-label">Location of Workshop *</label>  
       <div class="col-md-4 inputGroupContainer">
 	<div class="input-group">
-	  <input type="text" name="location" ng-model="location" ng-pattern="/^[a-zA-Z\s,]*$/"  class="form-control"
-		 placeholder="College Name, City Name, State Name, Country Name" required>
+	  <input type="text" ng-mouseover="targetWorkshopLocation('over')" ng-mouseleave="targetWorkshopLocation('unover')" name="expts" min="0" ng-model="expts"  class="form-control"
+                 type="text" name="location" ng-model="workshops"
+		 ng-pattern="/^[a-zA-Z\s,]*$/" class="form-control"
+		 placeholder="College Name, City Name, State Name,
+		 Country Name" required>
 	  <span style="color:red" ng-show="myForm.location.$dirty && myForm.location.$invalid">
 	    <span ng-show="myForm.location.$error.required">Location is required.</span>
 	    <span ng-show="myForm.location.$error.pattern">Enter valid location</span>
-	  </span>
+	  </span> {{info1}}
 	</div>
       </div>
     </div>
 
 
     <div class="form-group">
-      <label class="col-md-4 control-label">Participating Institutes *</label>  
+      <label class="col-md-4 control-label">Workshop College Name *</label>  
       <div class="col-md-4 inputGroupContainer">
 	<div class="input-group">
-	  <input type="text" name="insts" ng-model="insts"  ng-pattern="/^[a-zA-Z._-\s,]*$/" class="form-control"
+	  <input type="text" ng-mouseover="targetWorkshopCollegeName('over')" ng-mouseleave="targetWorkshopCollegeName('unover')" name="parti" min="0" ng-model="parti"  class="form-control"
+                type="text"
+		 name="insts" ng-model="insts"
+		 ng-pattern="/^[a-zA-Z._-\s,]*$/" class="form-control"
```

Comment:
class="form-control" which is defined twice.

Reply:
Fixed.
https://github.com/vlead/outreach-portal/commit/2b00dbfc509e84680e8829026d7bfcb720903342